usaspending_api/accounts/v2/filters/account_download.py

```diff
@@ -133,8 +133,6 @@ def get_agency_name_annotation(relation_name: str, cgac_column_name: str) -> Sub
 
 def generate_treasury_account_query(queryset, account_type, tas_id):
     """ Derive necessary fields for a treasury account-grouped query """
-    # Derive treasury_account_symbol, allocation_transfer_agency_name, agency_name, and federal_account_symbol
-    # for all account types
```

Comment:
Did this change, or did you feel it was just an unnessicary comment?

Reply:
Subjectively didn't seem very useful.  The function comment block essentially says the same thing minus the actual derived field names which are listed out below.  I can add it back if you feel anything was lost with its removal.

Reply:
I just wanted to make sure this code wasn't somehow doing something different now. If it had been,  I would have wanted that documented.